setup.py

```diff
@@ -13,16 +13,16 @@
     version=__version__,
     description=('checks playbooks for practices and behaviour that could potentially be improved'),
     keywords='ansible, lint',
```

Comment:
this should be a list

Reply:
(And if you check the current PyPI page, it only has `ansible` keyword on it, confirming that it doesn't work with a raw python string)